### PR TITLE
Fix for duplicate gesture not copying what you expect.

### PR DESCRIPTION
**Pull request by sfraser**
Brah. Fixxored the Duplicate Bug.

### CODE REVIEW

**Comment by sfraser** [comment]
Oops also BTW I turned back on some Delete code that was crashing cause it wasn't there.

**Comment by sfraser** [comment]
Do u care?

File: TestingApp/Projects/UITableView/MyTableViewController.swift [hunk 1]
```diff
@@ -67,7 +67,7 @@ class MyTableViewController: UITableViewController {
     override func tableView(_ tableView: UITableView,
                             leadingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
         let duplicateAction = UIContextualAction(style: .normal, title: "Duplicate") { _, _, done in
-            self.importantData.append(self.importantData[indexPath.item])
+            self.importantData.insert( self.importantData[indexPath.item], at: indexPath.item)
```
Comment (perryprog) [inline TestingApp/Projects/UITableView/MyTableViewController.swift]: Why is there a space here?

Comment (sfraser) [inline TestingApp/Projects/UITableView/MyTableViewController.swift]: oops that is lame hold please

**Pull request merged**